features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag

```diff
@@ -242,27 +242,28 @@ var loadTokens = function () {
         applicationId:application_id
         }, function (result) {
             if (result.error == false) {
-	       console.log(result);
-	       if (selected_type == "production") {
-		     if (result.apikeys.length > 0) {
-			token = result.apikeys[0].PRODUCTION_KEY;
-		        $("#access_token").val(token);
-		     } else {
-			$("#access_token").val("");
-		     }
-	       } else if (selected_type == "sandbox") {
-		     if (result.apikeys.length > 1) {
-			token = result.apikeys[1].SANDBOX_KEY;
-		        $("#access_token").val(token);
-		     } else {
-			$("#access_token").val("");
-		     }
-	       }
-	       if(token && token.trim() != "") {
+            if (selected_type == "production") {
```

Comment:
looks like the alignment is not correct. please verify.

Reply:
Fixed in commit 7d9d783ff33a7893e835d34e0098ac5da6afef37.